src/server/pegasus_server_impl.cpp

```diff
@@ -2695,6 +2699,22 @@ void pegasus_server_impl::update_user_specified_compaction(
     }
 }
 
+void pegasus_server_impl::update_allow_ingest_behind(const std::map<std::string, std::string> &envs,
+                                                     bool default_value)
+{
+    const auto &iter = envs.find(ROCKSDB_ALLOW_INGEST_BEHIND);
+    if (iter == envs.end()) {
+        _allow_ingest_behind = false;
```

Comment:
Why didn't you set it to default_value?

Reply:
If `ROCKSDB_ALLOW_INGEST_BEHIND` can't be found in app_envs, it should be set as false.
`default_value` only be used when app ROCKSDB_ALLOW_INGEST_BEHIND is invalid bool, we set it as default_value. 
If the db exists before, default_value is the lastest option value, if not, default_value is false. I will update this function comment.